package.json

```diff
@@ -37,7 +39,8 @@
     "karma": "^6.3.4",
     "postcss": "^8.3.9",
     "rollup": "^2.58.0",
-    "sass": "^1.43.2"
+    "sass": "^1.43.2",
+    "tailwindcss": "^2.2.19"
```

Comment:
It might make sense to make this an optional peer dependency for downstream projects that don't use Tailwind. This is done in the `peerDependenciesMeta` section - https://classic.yarnpkg.com/en/docs/package-json#toc-peerdependenciesmeta. Additionally you'll need to put any optional functionality under a separate entry point or use dynamic import to load it.

Example of making Tailwind optional:

```diff
diff --git a/lib/sass.js b/lib/sass.js
index e07139e..723f098 100644
--- a/lib/sass.js
+++ b/lib/sass.js
@@ -4,7 +4,6 @@ import { basename, dirname, extname } from 'path';
 import autoprefixer from 'autoprefixer';
 import postcss from 'postcss';
 import sass from 'sass';
-import tailwindcss from 'tailwindcss';
 
 /**
  * @typedef {import('tailwindcss/tailwind-config').TailwindConfig} TailwindConfig
@@ -34,6 +33,14 @@ export async function buildCSS(
   const minify = process.env.NODE_ENV === 'production';
   await mkdir(outDir, { recursive: true });
 
+  /** @type {typeof import('tailwindcss')} */
+  let tailwindcss;
+  try {
+    tailwindcss = (await import('tailwindcss')).default;
+  } catch {
+    // Ignored
+  }
+
   await Promise.all(
     inputs.map(async input => {
       const output = `${outDir}/${basename(input, extname(input))}.css`;
@@ -46,10 +53,12 @@ export async function buildCSS(
         sourceMap: sourcemapPath,
       });
 
-      const cssProcessor = postcss([
-        tailwindcss(tailwindConfig),
-        autoprefixer(),
-      ]);
+      const optionalPlugins = [];
+      if (tailwindcss) {
+        optionalPlugins.push(tailwindcss(tailwindConfig));
+      }
+
+      const cssProcessor = postcss([...optionalPlugins, autoprefixer()]);
 
       const postcssResult = await cssProcessor.process(sassResult.css, {
         from: input,
diff --git a/package.json b/package.json
index ec7ef8a..c82f751 100644
--- a/package.json
+++ b/package.json
@@ -42,6 +42,11 @@
     "sass": "^1.43.2",
     "tailwindcss": "^2.2.19"
   },
+  "peerDependenciesMeta": {
+    "tailwindcss": {
+      "optional": true
+    }
+  },
   "prettier": {
     "arrowParens": "avoid",
     "singleQuote": true
```